assets/js/data/validation/test/selectors.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Internal dependencies
+ */
+import {
+	getValidationErrorId,
+	getValidationError,
+	hasValidationErrors,
+} from '../selectors';
+import type { FieldValidationStatus } from '../reducers';
```

Comment:
We have an import error: Module '"../reducers"' has no exported member 'FieldValidationStatus'. Let's import it from the types.ts file instead.